ansible_ai_connect/organizations/models.py

```diff
@@ -23,20 +23,19 @@
 
 class Organization(models.Model):
     id = models.IntegerField(primary_key=True)
-    _telemetry_opt_out = models.BooleanField(default=False, db_column="telemetry_opt_out")
+    telemetry_opt_out = models.BooleanField(default=False, db_column="telemetry_opt_out")
+    extra_kwargs = {
+        "telemetry_opt_out": {"write_only": True},
```

Comment:
I don't understand why you're adding the `extra_kwargs` for `write_only: True`. Can you explain?

Reply:
It was to turn the flag read-only as a safeguard and be sure we only use the has_ property instead.

Reply:
https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields

Reply:
Actually, ignore what I just said. I was missing the serializer and the model...